dgraph/cmd/zero/http.go

```diff
@@ -231,9 +234,51 @@ func (st *state) getState(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) zeroHealth(ctx context.Context) (*api.Response, error) {
```

Comment:
I was looking at how alpha's health endpoint implemented and now I understood why there was an `all` arg and we were returning a list. We could do something similar here too, it'd be nice to have it behave similar to how alpha's endpoint is returning. It would also be useful to know what this node thinks of the health of other zero nodes. We will have to look into if we have access to health of other zero nodes.